tests/flows/block_hash_publish_receive.go

```diff
@@ -4,14 +4,16 @@ import (
 	"context"
 	"math/big"
 
+	coreEthClient "github.com/ava-labs/coreth/ethclient"
 	"github.com/ava-labs/subnet-evm/accounts/abi/bind"
 	"github.com/ava-labs/teleporter/tests/interfaces"
 	"github.com/ava-labs/teleporter/tests/utils"
 	. "github.com/onsi/gomega"
 )
 
 func BlockHashPublishReceive(network interfaces.Network) {
-	subnetAInfo, subnetBInfo, _ := utils.GetThreeSubnets(network)
+	subnetAInfo := network.GetPrimaryNetworkInfo()
```

Comment:
nit: can you rename these to primarySubnetInfo or similar

Reply:
same for other test cases

Reply:
"Primary Network" is actually the correct terminology: https://docs.avax.network/learn/avalanche/avalanche-platform